ROADMAP.md

```diff
@@ -0,0 +1,98 @@
+Ansible Roadmap
+=============
+This document is now the location for published Ansible Core roadmaps.     
+
+The roadmap will be updated by version. Based on team and community feedback, an initial roadmap will be published for a major or minor version (2.0, 2.1).  Subminor versions will generally not have roadmaps published.
+
+This is the first time we've published this and asked for feedback in this manner.  So feedback on the roadmap and the new process is quite welcome.  We are aiming for further transparency and better inclusion of both community desires and submissions.  
+
```

Comment:
Nitpicky docs comments... I try to avoid using "we" as a voice in technical documentation. I'd suggest:
"This is the first time Ansible has published this..."  and   "Ansible is aiming for further transparency..."